### DISCUSSION TITLE
Config Problems: Documentation seems to be outdated

### DISCUSSION THREAD

**Discussion by MilesAheadAlso** [fuatakgun/eufy_security#572] [Q&A] [answered]
I had everything working, but wanted to start from scratch.

This is the config page I see in HA. There is no mention of the `TRUSTED_DEVICE_NAME` in the documentation. Have I set this value correctly?
![image](https://user-images.githubusercontent.com/32249192/207031093-12f811d9-1ab9-4113-8f89-ef1a7e6a2b59.png)

**Comment by fuatakgun** [comment, marked as answer]
fixed in Alpha version
